modules/tinymce/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Contents would not be removed from the drag start source if dragging and dropping internally into a transparent block element. #TINY-9960
 - In some cases pressing enter would scroll the entire page. #TINY-9828
 - The border styles of a table were incorrectly split into a longhand form after table dialog updates. #TINY-9843
+- An "Uncaught TypeError: Cannot read properties of null" error would sometimes be thrown when updating the content of a `streamContent: true` iframe dialog component. #TINY-10128
```

Comment:
The error also shows up in the UI as per our design. I think it would be best to be specific on the type of error that shows up in case people actively search for that in the changelog.

Reply:
Only if the dialog catches the error; TinyMCE won't show the error visually itself.

Remember the boundary between what we do, and what integrators can do in their own code.